qdrant-landing/content/documentation/examples/hybrid-search-llamaindex-jinaai.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Chat With Product PDF Manuals Using Hybrid Search 
+weight: 27
+preview_image: /blog/hybrid-cloud-llamaindex/hybrid-cloud-llamaindex-tutorial.png
```

Comment:
@trean does this work? I need to override the regular doc image with our custom made one.